test/server/cards/03_TWI/units/RoyalGuardAttache.spec.ts

```diff
@@ -0,0 +1,24 @@
+describe('Royal Guard Attaché', function () {
+    integration(function (contextRef) {
+        describe('Royal Guard Attaché\'s ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['royal-guard-attache'],
+                    },
+                    player2: {
+                        groundArena: ['death-trooper'],
+                    },
+                });
+            });
+
+            it('should take 2 damages when played', function () {
```

Comment:
nit: if there is only one test case for a setupTest() we can remove the beforeEach and put the setup directly in the test case for readability, see Zillo Beast tests for an example